# request
string name
---
# response
float32[] poses
float32[] jacobians
